conf/application/test.conf

```diff
@@ -0,0 +1,8 @@
+application.mode=test
+db.default.url="jdbc:h2:mem:play;MODE=PostgreSQL;DB_CLOSE_DELAY=-1"
+db.default.driver="org.h2.Driver"
+db.default.username="sa"
+db.default.password=""
+db.default.jndiName=DefaultDS
+
+play.evolutions.autoApply=true
```

Comment:
this line needs to be in the dev configuration otherwise evolutions won't be applied automatically when you run your application locally (you'll see a "click to run evolutions" button if navigating to localhost:9000

Please add it to dev.conf